examples/asr/experimental/audio_processing/conf/multichannel_enhancement.yaml

```diff
@@ -0,0 +1,118 @@
+name: "multichannel_enhancement"
+
+model:
+  sample_rate: 16000
+  skip_nan_grad: false
+  num_outputs: 1
+
+  train_ds:
+    manifest_filepath: ???
+    input_key: audio_filepath
+    target_key: target_filepath
+    target_channel_selector: 0 # Target is the first channel from files in target_key
+    audio_duration: 4.0 # In seconds, audio segment duration for training
+    random_offset: True
+    min_duration: ${model.train_ds.audio_duration}
+    batch_size: 32
+    shuffle: true
+    num_workers: 8
+    pin_memory: true
+
+  validation_ds:
+    manifest_filepath: ???
+    input_key: audio_filepath
+    target_key: target_filepath
+    target_channel_selector: 0
+    batch_size: 1
+    shuffle: false
+    num_workers: 4
+    pin_memory: true
+
+  test_ds:
+    manifest_filepath: ???
+    input_key: audio_filepath
+    target_key: target_filepath
+    target_channel_selector: 0
+    batch_size: 1
+    shuffle: false
+    num_workers: 4
+    pin_memory: true
+
+  encoder:
+    _target_: nemo.collections.asr.modules.audio_modules.AudioToSpectrogram
```

Comment:
let's stick to NeMo ASR convention and call feature extractor (here `AudioToSpectrogram`) as `preprocessor` rather than `encoder`.